### PR TITLE
[IMP] open_academy: create test for the models and create a yml file T#52900

### PR DESCRIPTION
**Pull request by Jerudiel**
Testing log
Screenshot_20220128_111910

### CODE REVIEW

**Comment by Jerudiel** [comment]
@goliveirab could you review this? please

File: open_academy/tests/test_global_openacademy_course.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+from psycopg2.errors import UniqueViolation
+
+from odoo.tests.common import TransactionCase
+from odoo.tools import mute_logger
+
+
+class GloabalTestOpenAcademy(TransactionCase):
+    def setUp(self):
+        super().setUp()
+        self.variable = 'hello word'
```
Comment (goliveirab) [inline open_academy/tests/test_global_openacademy_course.py:11]: This is not used... you can remove this.

File: open_academy/tests/test_global_openacademy_course.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    def test_01_same_name_description(self):
+        with self.assertRaisesRegex(
+            UniqueViolation,
+            'duplicate key value violates unique constraint "course_title_unique"'
+        ):
+            self.create_course('test', 'test_description_00', None)
+            self.create_course('test', 'test_description_01', None)
+
+    def test_02_duplicate_course(self):
+        course = self.env.ref('open_academy.course_demo1')
```
Comment (goliveirab) [inline open_academy/tests/test_global_openacademy_course.py:41]: You could try use assertRecordValues, to check the expected values

**Comment by Jerudiel** [comment]
@goliveirab I made changes that you suggested, could you review this? Please

**Pull request merged**